### DISCUSSION TITLE
HTML

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#16] [Progress Tracker]
- Introduction to HTML?
- Creating HTML documents.
- Structure of an HTML document.
- HTML 4 and HTML 5.
- HTML 5 tags:
    - html
    - head
    - title
    - meta
    - body
    - p
    - h1 - h6
    - br
    - pre
    - hr
    - div
    - section
    - aside
    - code
    - mark
    - abbr
    - b
    - i
    - u
    - strong
    - del
    - img
    - a
    - table
    - tr
    - td
    - thead
    - tbody
    - tfoot
    - caption
    - audio
    - video
    - style
    - script
    - iframe
    - form
    - input
        - text
        - email
        - url
        - date
        - datetime-local
        - time
        - search
        - hidden
        - select: option, optgroup
        - password
        - tel
        - checkbox
        - radio
        - color
        - range
        - file
        - submit
        - reset
        - button

**Comment by Nandana-04** [comment]
Completed

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by riya45810** [comment]
Completed.

**Comment by akhilamp123** [comment]
Completed

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by ManikuttyBabu** [comment]
completed

**Comment by AshinDamodaran** [comment]
Completed

**Comment by Shamla-P** [comment]
Completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by kavitha-BS** [comment]
Completed

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by SivapriyaVB** [comment]
Completed
